lib/middleware/request_tracker.rb

```diff
@@ -599,6 +599,21 @@ def self.extract_view_tracking_data(env, status, headers)
       deferred_track_view: deferred_track_view,
       implicit_track_view: implicit_track_view,
       browser_page_view: browser_page_view,
+      tracking_url: env["HTTP_DISCOURSE_TRACKING_URL"],
+      tracking_referrer: env["HTTP_DISCOURSE_TRACKING_REFERRER"],
+      tracking_session_id: env["HTTP_DISCOURSE_TRACKING_SESSION_ID"],
+      user_agent: env["HTTP_USER_AGENT"],
```

Comment:
I do feel like we should centralize extracting addition tracking request headers somewhere so we are not extracting request headers here and also in the following spot:

https://github.com/discourse/discourse/blob/58a8668d047ba3e93e709570b6163a2a9d7da495/lib/middleware/request_tracker.rb#L193-L198